IliasDownloaderUniMA/__init__.py

```diff
@@ -127,6 +173,44 @@ def addCourses(self, *iliasids):
 			self.addCourse(iliasid)
 
 
+	def addAllUserCourses(self, excluded_ids=[]):
```

Comment:
```suggestion
	def addAllCurrentSemesterCourses(self):
```

Reply:
I don't fully get why you don't want the excluded_ids parameter.
It'd doesn't add that much complexity, but makes the method much more flexible.
First use case that I could think off: Some prof inserted another inconsistent, haunted structure in their course that breaks the parser. Course could be temporarily disabled this way til someone fixed it.

Reply:
> I don't fully get why you don't want the excluded_ids parameter.

I wasn't sure if you were aware of the problems withs `extractUserCourses()` and indented the user to exclude all unwanted ids. Now I see your point, so let's keep the `exlucuded_ids`. But `None` as default value is more consistent IMHO.